package.json

```diff
@@ -43,22 +43,25 @@
     "postcss-preset-env": "^6.7.0",
     "postcss-safe-parser": "^4.0.1",
     "regenerator-runtime": "^0.13.3",
-    "sass-loader": "7.3.1",
-    "style-loader": "^1.0.0",
+    "sass-loader": "^8.0.0",
```

Comment:
> yea, but webpacker in **default configuration does not use those parameters** that were changed...We can also update and loaders

This will break things for users that **chose to use those parameters** since we can only update loaders in our own codebase. We could have changelog entry that informs users how to migrate their `environment.js`, but this would need to be a breaking change.